docs/content/preview/admin/yb-admin.md

```diff
@@ -1862,6 +1862,43 @@ yb-admin \
 * *safe_time_lag_sec*: Safe time lag is computed as `(current time - current safe time)`.
 * *safe_time_skew_sec*: Safe time skew is computed as `(safe time of most caught up tablet - safe time of laggiest tablet)`.
 
+#### wait_for_replication_drain
+
+Verify when the producer and consumer are in sync for a given list of `stream_ids` at a given timestamp. 
+
+**Syntax**
+
+```sh
+yb-admin \
+    -master_addresses <target_master_addresses> \
+    wait_for_replication_drain \
+    <comma_separated_list_of_stream_ids> [<timestamp> | minus <interval>]
+```
+
+* *target_master_addresses*: Comma-separated list of target YB-Master hosts and ports. Default value is `localhost:7100`.
+* *comma_separated_list_of_stream_ids*: Comma-separated list of stream IDs.
+* *timestamp*: The time to which to wait for replication to drain. If not provided, it will be set to current time in the YB-Master API.
+* *minus <interval>*: The `minus <interval>` is the same format as in <a href="{{< relref "../explore/cluster-management/point-in-time-recovery-ycql.md" >}}">PITR documentation</a>, or see `yb-admin restore_snapshot_schedule` command).
+
+**Example**
+
+```sh
+./bin/yb-admin \
+    -master_addresses 127.0.0.11:7100,127.0.0.12:7100,127.0.0.13:7100 \
```

Comment:
not sure what we use elsewhere in this doc page, but since this needs to be run on the source, we may want this to be 127.0.0.1 / 2 / 3

Reply:
Fixed in https://github.com/yugabyte/yugabyte-db/pull/17328/commits/94ba04923068d2e3a081a80faeb511982957e35a